custom_components/tuya_local/devices/solaire_vitra_wifi_electric_panel_heater.yaml

```diff
@@ -0,0 +1,72 @@
+name: Sol*Aire Vitra WiFi Electric Panel Heater
+primary_entity:
```

Comment:
It would be useful to have a `products` section here with the product_id (or productKey) of your heater and the branding parts of the name.  The top level name can then be shortened to just the generic portion without redundant words like WiFi, in case other brands also match (such as rebadged heaters sold elsewhere in the world).

Reply:
Where would I find the product_id (if available)?  I can't find anything in the json from Download Diagnostics.
In the eu.iot.tuya.com Devices page, there is a column "Product" (between "Device ID" and "Source") with unpromising value "New-Solaire smart heater S2".  I'm guessing that might be products.name (can I have a name without an id?)?  The manual refers to "Vitra (s2)" so I think that might be the range of heaters.
This is my first Tuya device (that I'm using through Smart Things anyway -- always trying to stay local hence this integration!) so some guidance would be very welcome, thank you.

Reply:
The name does not have to be exactly what is in iot.tuya.com, but somewhere near that (in the same calls that give you local_key) there should be a product_id as well.  It is also visible if you do a network scan from tinytuya on the command line.

It is not in device diagnostics, because there is no known method to get this from a device that is connected locally. Only from the discovery broadcast messages and the cloud.

Reply:
I'll get round to having a look at using tinytuya soon.

Reply:
Added products.id, products.name and changed name and filename to match.